deno.jsonc

```diff
@@ -10,7 +10,7 @@
     }
   },
   "tasks": {
-    "test": "deno fmt --check && deno lint && deno test src",
-    "coverage": "deno test --coverage=.coverage src && deno coverage --exclude=fixtures --exclude=test .coverage"
+    "test": "deno fmt --check && deno lint && deno test -A src",
+    "coverage": "deno test -A --coverage=.coverage src && deno coverage --exclude=fixtures --exclude=test .coverage"
```

Comment:
Looking at this again, maybe we don't want to allow all permissions? Maybe we should keep the permissions in the `test` invocation the same as the permissions we grant hook invocations in a hermes app?

Reply:
K addressed in the latest commit. Resolving.